openstack/compute/v2/extensions/extendedstatus/results.go

```diff
@@ -0,0 +1,41 @@
+package extendedstatus
+
+type PowerState int
+
+type ServerExtendedStatusExt struct {
+	TaskState  string     `json:"OS-EXT-STS:task_state"`
+	VmState    string     `json:"OS-EXT-STS:vm_state"`
+	PowerState PowerState `json:"OS-EXT-STS:power_state"`
+}
+
+const (
+	NOSTATE = iota
+	RUNNING
+	_UNUSED1
+	PAUSED
+	SHUTDOWN
+	_UNUSED2
+	CRASHED
+	SUSPENDED
+)
+
+func (ps PowerState) String() string {
```

Comment:
I apologize, I should have caught this earlier. According to the [styleguide](https://github.com/gophercloud/gophercloud/blob/master/STYLEGUIDE.md#naming) we name all receivers `r`. So change `ps` to `r` here.

After that, this PR is good to go.